src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTrees.java

```diff
@@ -225,9 +217,8 @@ private void init(Context context) {
         parser = ParserFactory.instance(context);
         syms = Symtab.instance(context);
         fileManager = context.get(JavaFileManager.class);
-        JavacTask t = context.get(JavacTask.class);
-        if (t instanceof JavacTaskImpl taskImpl)
-            javacTaskImpl = taskImpl;
+        var task = context.get(JavacTask.class);
+        javacTaskImpl = (task instanceof JavacTaskImpl taskImpl) ? taskImpl : null;
```

Comment:
```suggestion
        javacTaskImpl = context.get(JavacTask.class) instanceof JavacTaskImpl taskImpl
                ? taskImpl : null;
```

Reply:
I don't see that as a significant improvement.